homepage.html

```diff
@@ -0,0 +1,112 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <!-- navbar -->
+ <nav class="navbar">
+    <div class="logo_item">
+      <i class="bx bx-menu" id="sidebarOpen"></i>
+      <img src="images/logo.png" alt=""></i>CREX
+    </div>
+    <div class="navbar_content">
+      <i class="bi bi-grid"></i>
+      <div class="zoom-out-preview">
+    
+       
+<button onclick="listAllImages();" id="logo">Click</button>
+<div id="ath-image-list" ></div>
+        <button type="button" class="btn btn-primary btn-lg"onclick="saveDesign()">Save</button>
+      <button type="button" class="btn btn-outline-dark btn-lg"onclick="previewZoomedOut()"s>Preview</button>
+    </div>
+      
+    </div>
+  </nav>
+
+
+  <nav class="sidebar">
+    <div class="menu_content">
+      <ul class="menu_items">
+        <div class="menu_title menu_dahsboard"></div>
+          <li class="item">
+          <div href="#" class="nav_link submenu_item">
+            <span class="navlink_icon">
+              <i class="bx bx-grid-alt"></i>
+            </span>
+            <span class="navlink">Category</span>
+            <i class="bx bx-chevron-right arrow-left"></i>
+          </div>
+
+          <ul class="menu_items submenu">
+            <a href="#" class="nav_link sublink">Tshirt</a>
+            <a href="#" class="nav_link sublink">Bottle</a>
+            <a href="#" class="nav_link sublink">Cap</a>
+            <a href="#" class="nav_link sublink">Bag</a>
+          </ul>
+        </li>
+        <!-- end -->
+      </ul>
+
+      <ul class="menu_items">
+        <div class="menu_title menu_editor"></div>
+        <!-- duplicate these li tag if you want to add or remove navlink only -->
+        <!-- Start -->
+        <li class="item">
+          <a href="#" class="nav_link">
+            <span class="navlink_icon">
+              <i class="bx bxs-magic-wand"></i>
+            </span>
+            <span class="navlink" >Logos</span>
+            
+          </a>
+          
+          <!-- <div class="col custom-height-1 custom-shape-col ">
+            <img src="shapes\love_small_black.png"onclick="changeTshirtImage('shapes/love_small_black.png')">
+          </div> -->
+
+          <!-- <div class="col custom-height-1 custom-shape-col ">
+            <img src="shapes\love_small_black.png"onclick="changeTshirtImage('shapes/love_small_black.png')">
+          </div> -->
+          <!-- <div class="col custom-height-1 custom-shape-col ">
+            <img src="shapes\love_small_black.png"onclick="changeTshirtImage('shapes/love_small_black.png')">
+          </div> -->
```

Comment:
Remove Unnecessary comments.

Reply:
I have made the changes .Please check it again

Reply:
okay.Thankyou